termenv_unix.go

```diff
@@ -8,10 +8,16 @@ import (
 	"os"
 	"strconv"
 	"strings"
+	"time"
 
 	"golang.org/x/sys/unix"
 )
 
+const (
+	// timeout for OSC queries
+	oscTimeout = 5 * time.Second
```

Comment:
maybe make this configurable?

Reply:
Since there's no real `termenv` instance you initialize, I guess I could just make it public? Not the best design decision if you look at the `net/http` package, but unlike it there's only ever one terminal session/instance, so it kind of is a global setting.

What do you think?

Reply:
yup, sounds good!